### ISSUE TITLE
HA Cluster ERR_SSL_PROTOCOL_ERROR

### ISSUE THREAD

**Issue by marcstreeter** [rancher/rancher#20317]
**What kind of request is this (question/bug/enhancement/feature request):**
question or bug? I'm not sure if I'm doing something wrong with my HA install

**Steps to reproduce (least amount of steps as possible):**
- clean install of Ubuntu 18.04 (with LVM enabled) on 3 machines
- these [exact steps](https://github.com/marcstreeter/kubewhistle/blob/master/README.rke.md) 

**Result:**
Originally this worked...now, with no changes that I'm aware of I always get `ERR_SSL_PROTOCOL_ERROR`.

**Other details that may be helpful:**
this is [exactly how I prepared each node](https://github.com/marcstreeter/kubewhistle/blob/master/README.node.md) via RKE

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): I used `helm repo add rancher-stable https://releases.rancher.com/server-charts/stable`
- Installation option (single install/HA): HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): RKE
- Machine type (cloud/VM/metal) and specifications (CPU/memory): Baremetal
- Kubernetes version (use `kubectl version`): v1.13.5

```
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.1", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"clean", BuildDate:"2019-04-19T22:12:47Z", GoVersion:"go1.12.4", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.5", GitCommit:"2166946f41b36dea2c4626f90a77706f426cdea2", GitTreeState:"clean", BuildDate:"2019-03-25T15:19:22Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):

```
Client:
 Version:           18.09.6
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        481bc77
 Built:             Sat May  4 02:35:57 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.6
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.8
  Git commit:       481bc77
  Built:            Sat May  4 01:59:36 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```

**Comment by marcstreeter** [comment]
Seems similar to his issue, but I was unsuccessful with his steps https://github.com/rancher/rancher/issues/17906

**Comment by marcstreeter** [comment]
sorry should I give more info?

**Comment by loganhz** [comment]
@marcstreeter 
This issue has fixed in 2.2.4 
Please try again. Thanks
